File: src/main/java/com/objects/CommanderDamage.java
package com.objects;

public class CommanderDamage {
    private String player, enemyPlayer, commander;
    private int damage;
    public CommanderDamage() {

    }
    public String getPlayer() {
        return player;
    }

    public void setPlayer(String player) {
        this.player = player;
    }

    public String getEnemyPlayer() {
        return enemyPlayer;
    }

    public void setEnemyPlayer(String enemyPlayer) {
        this.enemyPlayer = enemyPlayer;
    }

    public String getCommander() {
        return commander;
    }

    public void setCommander(String commander) {
        this.commander = commander;
    }

    public int getDamage() {
        return damage;
    }

    public void setDamage(int damage) {
        this.damage = damage;
    }


}

File: Backend/MTGCollectiveLifeCounterBackend/Models/CommanderDamage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MTGCollectiveLifeCounterBackend.Models {
    public class CommanderDamage {
        public string Player { get; set; }
        public string EnemyPlayer { get; set; }
        public string Commander { get; set; }
        public int Damage { get; set; }
    }
}
